pkg/operation/botanist/botanist.go

```diff
@@ -210,6 +211,7 @@ func New(ctx context.Context, o *operation.Operation) (*Botanist, error) {
 	if err != nil {
 		return nil, err
 	}
+	o.Shoot.Components.HVPA = hvpa.New(nil, b.Shoot.SeedNamespace, hvpa.Values{})
```

Comment:
Why are we passing nil here, for the client?

Reply:
HVPA is a seed system component and there is no dedicated instance for shoots. We only add it here so that we can access the methods of the `component.MonitoringComponent` interface. For those, no client is needed.